scripts/install-chef-suse.sh

```diff
@@ -130,7 +131,7 @@ pipe_show_and_log () {
         rm -f $t
         dialog --clear >&3
     fi
-    tee -a /dev/fd/3 /dev/fd/4 > /dev/null
+    tee -a /dev/fd/4 >&3
```

Comment:
I deployed a M7 with only this patch and it successfully installed crowbar.
I can not explain the cause of the bug either, but I verified that this fixes this bug.